gips/0083.md

```diff
@@ -0,0 +1,240 @@
+usage:
+  serviceType: substreams-v1
+  network: mainnet
+  nonce: some-uuid
+```
+
+then allocating to it, sharing the allocation ID with the Payment Gateway, in order for the Indexer to receive payment. Mind you, this isn't the same thing as the _Substreams Service Deployment_ manifest, as this one is short-lived as to not be exposed to the curation fee market.
+
+This flow is expected to be simplified in the future, with the Payment Gateway being able to automatically detect new Substreams services, and allocate to them, without manual intervention.
+
+Future opportunities would include self-serve APIs for Indexers to claim payment and do the dance automatically. Thus an Indexer would be able to lower the risks of not receiving payments from the Gateway.
+
+Eventually, TAP could be used to send payments alongside blockchain data payloads (BlockScopedData), or some other trust-minimized payment mechanism.
+
+
+## Pricing
+
+Pricing is currently outside of the scope of this proposal. The protocol (contracts) itself does not provide pricing mechanisms, so this responsibility is passed to the Payment Gateway.
+
+The Payment Gateway MUST transfer funds to the Indexer according to a Pricing Period. Ideally this period is the shortest possible, to reduce counterparty risks.
+
+
+### Current implementation and future opportunities
+
+THIS SECTION IS NON-NORMATIVE
+
+Current pricing is set at a fixed price, defined on The Graph Market (https://thegraph.market). It has been measured by the Core Devs to have unit economics that makes sense for operators, and will be revised as the system matures.
+
+Consumption is measured in terms of Terabytes read from the backing stores and/or Egress bytes. Fees are cumulated during a period of time, until payment is settled. This time period will start at 1 month and be adjusted to become lower and lower as the system matures, to address counterparty risks.
+
+Although initial integration starts with a simple pricing strategy, the integration of Substreams on The Graph Network by no means limits the future opportunities to adjust and augment the business models and pricing strategies.
+
+Future GIPs will be proposed to enable more complex pricing strategies, such as pay-per-cpu-time, pay-per-storage, pay-per-query, pay-per-block, pay-per-attestation, etc.
+
+### Curation
+
+THIS SECTION IS NON-NORMATIVE
+
+Curation can happen on the published Subgraph manifest above, although a nonce may be added to avoid paying curation fees. Because there are no indexer rewards, curation is not adding value, therefore it is reasonable for participants to want to avoid it.
+
+
+
+## Economic Security and Disputes
+
+Economic security is achieved through slashing and disputes, similar to subgraphs as of Jan 24th 2025. Indexers providing incorrect data CAN have their stake slashed. Therefore, to be an Indexer recognized by the Payment Gateway, one must staked the Network token, just like for subgraphs.
```

Comment:
Has there been discussions on setting the minimum staking requirement for substreams or the same as for subgraphs? Any clarifications on the slashing penalties?